Workflow para envio usando API do Chatwoot

lucascs1 opened the pull request:
Criei um novo workflow alterando o antigo para que os envios sejam feitos usando a API do próprio chatwoot.
Dessa forma é possível usar qualquer canal de (ou api) que os disparos serão feitos normalmente.

### Detalhes do workflow

No começo do workflow já ficam os nós para incluir as informação do chatwoot e também do contato que mandar as informações de disparo.
![image](https://github.com/ssteeltm/chatwootBrasil/assets/123115563/b5dcb3b3-b673-440f-9253-1a9e15c069a5)

> Dentro do nó **Info Chatwoot** o campo _user_token_ é Token de acesso que fica na fim da página de configuração do perfil do usuário admin (o mesmo utoken da quepasa).


> A validação do inbox sms usado para os disparos agora só leva em consideração o **nome** do caixa de sms. Os demais dados como número de telefone não são mais considerados.
> ![image](https://github.com/ssteeltm/chatwootBrasil/assets/123115563/fb27dc63-4821-420b-b789-49069aca8a06)


### Alteração na criação da campanha
Qualquer canal pode ser usado para disparar as mensagens da campanhas usando o inbox_sms
![image](https://github.com/ssteeltm/chatwootBrasil/assets/123115563/2bd89f3d-7d90-4478-90ff-c89475fa70b3)

> Por conta disso é obrigatório expecificar no título da campanha o nome do inbox que irá disparar  campanha.


Versão do n8n utilizada 1.31.1 - _versõe abaixo da 1.3 podem ter problemas com o node set._

lucascs1 committed:
Add files via upload

Refiz o workflow do n8n para disparar as mensagens usando a api do proprio chatwoot ao invés da quepasa.

lucascs1 committed:
Update Campanha_CW_API_Chatwoot.json

ssteeltm commented:
Excelente ideia! É o mesmo meio que uso na minha integração do typebot.

Vamos só testar e aprovar!

lucascs1 committed:
Update Campanha_CW_API_Chatwoot.json

Correção nos erros - query de listagem dos contatos e envio de msg de informações

lucascs1 committed:
Delete Campanha_CW_API_Chatwoot.json
